### PR TITLE
Icons: comment for local path + deprecated icons

### PR DESCRIPTION
**Pull request by vvalentin-lucca**
## Description

-----

-----

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2337/storybook

**Review by jeremie-lucca — APPROVED** [review]

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/7693/ui-diff)

**Review by shajz — APPROVED** [review]

File: packages/icons/update-icons.js [hunk 1]
```diff
@@ -39,7 +39,9 @@ const list = `${generatedWarning}export const IconsList = [\n\t${icons
 
 writeFileSync(join(__dirname, './icons-list.ts'), list);
 
-const scssConfig = `${generatedWarning}$font-path: '//cdn.lucca.fr/lucca-front/icons/font/lucca-icons' !default;
+const scssConfig = `${generatedWarning}// to test locally (without the CDN)
+// $font-path: '../../font/lucca-icons' !default;
```
Comment (mbriand-lucca) [inline packages/icons/update-icons.js:43]: normal de garder en commentaire cette ligne ?

Comment (vvalentin-lucca) [reply packages/icons/update-icons.js:43]: Yes, it's not easy to recover this path.

**Pull request merged**